### PR TITLE
Updates to personal_sign and fix QR code paste emulator button

### PR DESCRIPTION
**Pull request by jinchung**
Fixes APP-769

### CODE REVIEW

**Review by dereknelson — APPROVED** [review]
lgtm

File: src/components/qrcode-scanner/EmulatorPasteUriButton.js [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
 import { useWalletConnectConnections } from '@/hooks';
+import { pair as pairWalletConnect } from '@/walletConnect';
+import { parseUri } from '@walletconnect/utils';
 
 export default function EmulatorPasteUriButton() {
   const { result: isEmulator } = useIsEmulator();
   const { walletConnectOnSessionRequest } = useWalletConnectConnections();
   const { colors } = useTheme();
 
   const handlePastedUri = useCallback(
```
Comment (skylarbarrera) [inline src/components/qrcode-scanner/EmulatorPasteUriButton.js:15]: ive never used this in my life 😆

File: src/parsers/requests.js [hunk 2]
```diff
@@ -62,10 +63,14 @@ export const getRequestDisplayDetails = (
     let message = payload?.params?.find(p => !isAddress(p));
     try {
       if (isHexString(message)) {
-        message = convertHexToUtf8(message);
+        message = toUtf8String(message);
```
Comment (skylarbarrera) [inline src/parsers/requests.js:66]: is this functionality different? if this is the root of the issue we should bring this up to walletconnect

**Review by skylarbarrera — APPROVED** [review]
👍

**Pull request merged**